iso15118/secc/states/iso15118_2_states.py

```diff
@@ -1950,7 +1950,7 @@ async def process_message(
         msg = self.check_msg_v2(
             message,
             [PreChargeReq, PowerDeliveryReq],
-            not self.precharge_req_was_reveived,
+            not self.expecting_precharge_req,
```

Comment:
this now is confusing... I guess you meant to set the `expecting_precharge_req` to True, so that here we actually are checking for the presence of it and  not for the `not self.expecting_precharge_req`

Reply:
Yes you are right.